specification/databox/resource-manager/Microsoft.DataBox/preview/2021-08-01-preview/examples/AvailableSkusPost.json

```diff
@@ -1,20 +1,14 @@
 {
   "parameters": {
-    "subscriptionId": "fa68082f-8ff7-4a25-95c7-ce9da541242f",
-    "resourceGroupName": "bvttoolrg6",
+    "subscriptionId": "YourSubscriptionId",
+    "resourceGroupName": "YourResourceGroupName",
```

Comment:
@vaishnavi-janardhan Could you declare the context about the PR? I think the old examples are better since the specific fake value is easier to understand for users, especially for those who are not familiar with Azure.

Reply:
@msyyc, those were our actual test subscription values which we want to scrub from all example files.

Reply:
Since these example files are used for REST API doc generation, we are modifying them to include generic dummy data for all fields.